build-system/tasks/karma.conf.js

```diff
@@ -40,7 +40,17 @@ module.exports = {
   browserify: {
     watch: true,
     debug: true,
-    transform: ['babelify'],
+    transform: process.env.TRAVIS ? ['babelify'] : [
```

Comment:
(This is because even including this config seems to be causing sauce labs to run out of heap, and we had no near-term intention of making Travis run the coverage testing anyway)